docs/tutorials/tutorial_3k_tcr.md

```diff
@@ -507,38 +508,38 @@ ir.pl.spectratype(
 
 ### Repertoire overlaps
 
-```python
-df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
-```
+Overlaps in the adaptive immune receptor repertoire of samples or sample groups enables to pinpoint important clonotype groups, as well as to provide a measure of similarity between samples.  
+Running Scirpy's `repertoire_overlap` tool creates a matrix featuring the abundance of clonotypes in each sample. Additionally, it also computes a (Jaccard) distance matrix of samples as well as the linkage of hierarchical clustering.
 
 ```python
+df, dst, lk = ir.tl.repertoire_overlap(adata, "sample", inplace=False)
 df.head()
 ```
 
-```python
-ir.pl.repertoire_overlap(adata, "sample")
-```
+The distance matrix can be shown as a heatmap, while samples are reordered based on the linkage computed previously.
 
 ```python
 ir.pl.repertoire_overlap(adata, "sample", heatmap_cats=["patient", "source"])
 ```
 
-```python
-ir.pl.repertoire_overlap(adata, "sample", dendro_only=True, heatmap_cats=["source"])
-```
+A specific pair of samples can be compared on a scatterplot, where dot size corresponds to the number of clonotypes at a given coordinate.
 
 ```python
-ir.pl.repertoire_overlap(adata, "sample", pair_to_plot=["LN2", "LT2"])
+ir.pl.repertoire_overlap(adata, "sample", pair_to_plot=["LN2", "LT2"],fig_kws={"dpi": 120})
 ```
 
+### Discovering clonotypes preferentially occuring in a group
+
+Clonotypes associated with an experimental group (a given cell type, samle or diagnosis) might be important candidates as biomarkers or disease drivers. Scirpy offers `clonotype_imbalance` to rank clonotypes based on Fisher's exact test comparing the fractional presence of a given clonotype in two groups.
```

Comment:
This is more for me to understand, but maybe this could be added here as well: 
How would the contingency table look like that you use for fisher's test? 
Is this robust to clonotypes with just a single cell in one sample and no cell in the other?

Reply:
And can you maybe add a comment what the "normalized abundance" refers to in this case?

Reply:
Normalized abundance is the number of cells in the simplest case. If cell numbers in samples are very different, then this can be a fraction in a given sample or any other weighted value. But let us say, we stay with the simple case and let it be cell numbers for now and compare tumor samples to a control.
The contingency table would then be: [[nTc, nCc], [sumTc-nTc, sumCc-nCc]], where
- nTc is the number of cells in all Tumor samples that belong to clonotype X
- nCc is the number of cells in all Control samples that belong to clonotype X
- sumTc is the sum of cells in all Tumor samples
- sumCc is the sum of cells in all Control samples  
I would also want to know how robust this approach is. That was one of the reasons I termed this "experimental". The ratios are already very small if we take a look at total cell numbers and cells that belong to a given clonotype.  
But at least theoretically, Fisher's test can handle zero values.